tests/utils/test_shell_util.py

```diff
@@ -140,7 +139,8 @@ def test_run_command_should_raise_an_exception_when_the_command_fails(self):
             shellutil.run_command(command)
 
         exception = context_manager.exception
-        self.assertEquals(str(exception), "'ls' failed: 2")
+        self.assertIn("'ls' failed: 2", str(exception))
```

Comment:
btw - python 2.6 doesn't have an assert to match a regex, I need to add that to the test utilities.

i'll do that on a separate PR, in the meanwhile I split the check on 2 asserts